components/org.wso2.carbon.identity.application.authenticator.oidc/src/main/java/org/wso2/carbon/identity/application/authenticator/oidc/OIDCAuthenticatorConstants.java

```diff
@@ -34,6 +34,8 @@ private OIDCAuthenticatorConstants() {
     public static final String ACCESS_TOKEN = "access_token";
     public static final String ID_TOKEN = "id_token";
 
+    public static final String OIDC_LOGOUT_URL="OIDCLogoutEPUrl";
```

Comment:
Check formatting

Reply:
Better to move this to OIDCAuthenticatorConstants.IdPConfParams (along with USER_INFO_EP constant). Let's follow similar convention as well.

https://github.com/wso2-extensions/identity-outbound-auth-oidc/blame/b1148dad869615952ded22d5a3714aa029753ff3/components/org.wso2.carbon.identity.application.authenticator.oidc/src/main/java/org/wso2/carbon/identity/application/authenticator/oidc/OIDCAuthenticatorConstants.java#L64